src/workspace-hack/Cargo.toml

```diff
@@ -86,6 +86,8 @@ num-bigint = { version = "0.4" }
 num-integer = { version = "0.1", features = ["i128"] }
 num-iter = { version = "0.1", default-features = false, features = ["i128", "std"] }
 num-traits = { version = "0.2", features = ["i128", "libm"] }
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
cc @xxchan Is it okay to you?

Reply:
slightly prefer https://github.com/RustCrypto/block-ciphers

Reply:
will change to https://docs.rs/aes/latest/aes/ because openssl can make troubles when compiling

Reply:
replacing dep changes a lot to this impl, lets see if everything goes well